website/common/script/content/faq.js

```diff
@@ -1,4 +1,5 @@
 import t from './translation';
+import nconf from 'nconf';
```

Comment:
I don't think you can use nconf here, because it will not work. You need to include config variables on the window object if it is use in the client code: https://github.com/HabitRPG/habitica/blob/develop/website/server/middlewares/locals.js